packages/runtime/src/services/Runtime.ts

```diff
@@ -16,22 +17,21 @@ export default abstract class Runtime
 
     get url() { return this.#url; }
 
-    async import(url: string): Promise<Module>
+    set healthCheckFiles(filenames: Set<string>)
     {
-        return ModuleLoader.load(url);
+        this.#healthCheckFiles = filenames;
     }
 
-    async importHealthCheck(url: string): Promise<void>
-    {
-        const module = await this.import(url);
-        const healthCheck = module.default as HealthCheck;
+    abstract import(url: string, scope: ExecutionScope): Promise<Module>;
 
-        if (healthCheck?.isHealthy === undefined)
-        {
-            throw new InvalidHealthCheck(url);
-        }
+    start(): Promise<void>
+    {
+        return this.#importHealthChecks();
+    }
 
-        this.addHealthCheck(healthCheck as HealthCheck);
+    async stop(): Promise<void>
+    {
+        this.#clearHealthChecks();
     }
 
     addHealthCheck(healthCheck: HealthCheck): void
```

Comment:
This function is not called from external sources anymore. The function should be private to keep the code clean and simple.

Reply:
Testing the health check feature will become very difficult if we make this private. Also, it leaves the possibility open for manually adding health checks to the runtime, like the client. Besides all that, making it private doesn't do much for the simplicity of the code ;-)